lib/flag_shih_tzu.rb

```diff
@@ -18,6 +21,7 @@ def self.included(base)
   class IncorrectFlagColumnException < Exception; end
   class NoSuchFlagQueryModeException < Exception; end
   class NoSuchFlagException < Exception; end
+  class InvalidValueForFlagException < Exception; end
```

Comment:
Yeah, all of these classes should not be inheriting from `Exception`. 💯 Needs to be fixed.